services/learn-card-network/brain-service/src/helpers/notifications.helpers.ts

```diff
@@ -0,0 +1,97 @@
+interface IPushNotificationRequest {
+    to: string;
+    title: string;
+    message?: string;
+    url?: string;
+    actionType: string;
+}
+
+interface IPushRegistrationRequest {
+    profileId: string;
+    deviceToken: string;
+}
+
+export async function SendPushNotification(args: IPushNotificationRequest) {
+    try {
+        if (process.env.WEBHOOK_URL) {
+            const response = await fetch(process.env.WEBHOOK_URL, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'apikey': `${process.env.APIKEY}`,
+                },
+                body: JSON.stringify({
+                    actionType: args.actionType,
+                    profileId: args.to,
+                    data: {
+                        title: args.title,
+                        message: args.message,
+                        url: args.url,
+                    },
+                }),
+            });
+            if (!response.ok) {
+                throw new Error(`HTTP error! status: ${response.status}`);
+            }
+            const data = await response.json();
+            return data;
+        }
+    } catch (error) {
+        console.error('Error fetching data:', error);
+        console.log(error);
```

Comment:
Should we remove the normal console log?

```suggestion
        console.error('Notifications Helpers - Send Push Notification:', error);
```

Reply:
Should we log extra information here for easy debuggin?

Reply:
removed standard console.log, replaced with more helpful error